BridgeEmulator/HueEmulator3.py

```diff
@@ -505,7 +528,7 @@ def sendLightRequest(light, data):
                 if key == "on":
                     payload["5850"] = int(value)
                 elif key == "transitiontime":
-                    payload["transitiontime"] = value
```

Comment:
transitiontime is good here, because is used just to know if this flag is set. Later when coap client is executed this is converted to correct id

Reply:
forget, now i saw you sent everything in one command

Reply:
I've also changed the coap executing so it dumps the whole payload instead of iterating it. This would cause "transtiontime" to be included in the coap command since it was added in the payload. That's why I changed this line to add the transitiontime with the correct key.

Also currently I've added a default transition time just before firing the coap if it doesn't exists yet, but I'm still testing if I like it or not.

Reply:
And yes, everything works in 1 command :)